.travis.yml

```diff
@@ -18,75 +18,64 @@ cache:
 
 install:
 - grep -Erl git-lfs\|couchdb /etc/apt | sudo xargs rm -v
+- pip install -U pip==20.2.1
 - .ci/${MODE}_install.py
 
+# Travis has a 4MB log limit (https://github.com/travis-ci/travis-ci/issues/1382), but verbose Mitogen logs run larger than that
+# in order to keep verbosity to debug a build failure, will run with this workaround: https://stackoverflow.com/a/26082445
 script:
 - .ci/spawn_reverse_shell.py
-- .ci/${MODE}_tests.py
-
+- MODE=${MODE} .ci/travis.sh
 
 # To avoid matrix explosion, just test against oldest->newest and
 # newest->oldest in various configuartions.
 
 matrix:
-  allow_failures:
-    # Python 2.4 tests are still unreliable
-    - language: c
-      env: MODE=mitogen_py24 DISTRO=centos5
-
   include:
     # Debops tests.
-    # 2.9.6; 3.6 -> 2.7
-    - python: "3.6"
-      env: MODE=debops_common VER=2.9.6
-    # 2.8.3; 3.6 -> 2.7
-    - python: "3.6"
-      env: MODE=debops_common VER=2.8.3
-    # 2.4.6.0; 2.7 -> 2.7
-    - python: "2.7"
-      env: MODE=debops_common VER=2.4.6.0
+    # NOTE: debops tests turned off for Ansible 2.10: https://github.com/debops/debops/issues/1521
+    # 2.10; 3.6 -> 2.7
+    # - python: "3.6"
+    #   env: MODE=debops_common VER=2.10.0
+    # 2.10; 2.7 -> 2.7
+    # - python: "2.7"
+    #   env: MODE=debops_common VER=2.10.0
 
     # Sanity check against vanilla Ansible. One job suffices.
-    - python: "2.7"
-      env: MODE=ansible VER=2.8.3 DISTROS=debian STRATEGY=linear
```

Comment:
I would like to keep at least one CI job with `STRATEGY=linear`, as a reference to compare against.

Reply:
I migrated the linear strategy test to the azure tests section because travis doesn't let us have long-running jobs. It caps them at 50 min and the new Ansible version tests take like 1.5 hours on vanilla Ansible :( The comment I left points to a github comment explaining the 50 min limit, but I'll make the comment more clear 👍